### PR TITLE
[VOL-2733] Properly configuring ONOS logging in kind-voltha environment

### PR DESCRIPTION
**Pull request by teone**

### CODE REVIEW

File: full-values.yaml [hunk 1]
```diff
@@ -89,6 +90,9 @@ onos_env:
 - name: ONOS_APPS
   value: "drivers,openflow-base,hostprovider"
 
+kafka_logging:
+  brokers: ['voltha-kafka.voltha.svc:9092']
```
Comment (dbainbri-ciena) [inline full-values.yaml]: `.cluster.local`, which is how this is specified elsewhere in the file? Do we need to set this up one place and reference in others? and where does this setting go? ONOS? it is set at the top level?

Comment (teone) [reply full-values.yaml]: Yes, it's set at the top level of the `onos` chart: https://github.com/opencord/helm-charts/blob/master/onos/values.yaml#L33

Updated the patch with `.cluster.local`

File: minimal-values.yaml [hunk 2]
```diff
@@ -116,6 +117,9 @@ onos_env:
 - name: ONOS_APPS
   value: "drivers,openflow-base,hostprovider"
 
+kafka_logging:
+  brokers: ['voltha-kafka.voltha.svc:9092']
```
Comment (davidkbainbridge) [inline minimal-values.yaml]: and here

Comment (teone) [reply minimal-values.yaml]: Sorry, missed to hit `save` on one of the two...

**Comment by teone** [comment]
Loggin on kafka has been disabled by default with this patch: https://gerrit.opencord.org/#/c/18006/
this fix is not required anymore.

**Pull request closed without merging**